querybook/webapp/components/QueryRunButton/QueryRunButton.tsx

```diff
@@ -215,7 +215,7 @@ const QueryLimitSelector: React.FC<{
             customButtonRenderer={() => (
                 <div
                     className="flex-center ph4"
-                    aria-label="Only applies to select statements without limit"
+                    aria-label="Only applies to SELECT query without LIMIT"
```

Comment:
do we want to provide the option of disabling the limit feature, even the query doesn't have `LIMIT`?

Reply:
sounds good, in another PR? (since this one is getting a bit messy now)